src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -7,10 +7,12 @@ import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import akka.http.scaladsl.marshalling.ToResponseMarshallable
 import akka.http.scaladsl.model.StatusCodes.{BadRequest, Forbidden, NotImplemented, Unauthorized}
 import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
-import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server._
 import akka.http.scaladsl.unmarshalling.Unmarshal
 import akka.stream.ActorMaterializer
+import cats.instances.future._
+import cats.instances.list._
+import cats.syntax.traverse._
```

Comment:
I like these imports as much as anybody but are they used?  If not can we put in the compiler flag to enforce this going forward?